src/streaming/AudioConfig.py

```diff
@@ -0,0 +1,29 @@
+"""
+    Configuration for an audio stream
+"""
+
+class AudioConfig():
```

Comment:
If we support Audio and Video I think naming this audioconfig is misleading. Perhaps we should rename it MediaConfig